README.md

```diff
@@ -194,7 +203,7 @@ setTimestamp(Instant.now())
 
 - **flush**()
 
-Flushes the current MetricsContext to the configured sink and resets all properties, dimensions and metric values. The namespace and default dimensions will be preserved across flushes.
+Flushes the current MetricsContext to the configured sink and resets all properties and metric values. The namespace and default dimensions will be preserved across flushes. The default behavior is to preserve all custom dimensions as well, but this can be disabled by invoking `flushPreserveDimensions(false)`.
```

Comment:
could you put code snippet here as example

After flush,
1. Preserve customer dimensions + default dimensions
2. Only preserve default dimension
3. No dimension will be preserved

Reply:
Solved.